### ISSUE TITLE
BUG: BlanketConstantThicknessFP() with full covergae

### ISSUE THREAD

**Issue by RemiTheWarrior** [ukaea/paramak#86]
![image](https://user-images.githubusercontent.com/40028739/87774529-eedc1500-c824-11ea-8ff9-37ad45921dd1.png)

How to reproduce:

```python
from paramak import BlanketConstantThicknessFP

my_shape = BlanketConstantThicknessFP(100, start_angle=0, stop_angle=360, rotation_angle=360)
my_shape.solid

```

**Comment by Shimwell** [comment]
fixed by @RemiTheWarrior in the PR #89

**Comment by RemiTheWarrior** [comment]
@Shimwell This issue should be reopened as it hasn't been fixed by PR #89 and won't be in PR #90 ...
